pkg/cloudclient/gcp/private.go

```diff
@@ -208,13 +208,12 @@ func (c *Client) findGCPForwardingRuleForExtIP(rhapiLbIP string) (*compute.Forwa
 	if err != nil {
 		return nil, err
 	}
-	var fr *compute.ForwardingRule
 	for _, lb := range response.Items {
 		if lb.IPAddress == rhapiLbIP {
-			fr = lb
+			return lb, nil
```

Comment:
is `lb` typed appropriately here? I'm wondering why we historically did the reassignment if it was.